TopicGroup.sedona

```diff
@@ -0,0 +1,147 @@
+// Hey, emacs! -*- c++ -*-
+//
+// Copyright 2018 Andrey Skvortsov <andrej.skvortzov@gmail.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
BTW Apache 2 license is as open as MIT, but has patents protection for original authors. It shouldn't be a problem here, but who knows... 

IMHO it's important to have license statement with copyright holders in source file, so it's kept in source file even for permisive license like MIT in case the code is merged into other project. For consistency with all existed source files in the repo I can remove license statement from the header.